src/Components/Share/Share.tsx

```diff
@@ -38,12 +38,11 @@ const Share = forwardRef(function Share() {
     }),
   };
 
-  const shareUrl = getReferrer('shareLink');
-
+  const shareUrl = getReferrer('shareLink') || 'default';
```

Comment:
`getReferrer('shareLink')` should not be `undefined`. Also, this line means that if `getReferrer('shareLink') === undefined`, people would litterally be sharing the string "default".